defaults/defaults.clog_config

```diff
@@ -1,7 +1,7 @@
 {
   "MacroConfigurations": {},
   "Version": 1,
-  "CustomTypeClogCSharpFile": "../decoderTest/defaults.clog.cs",
+  "CustomTypeClogCSharpFile": "CLOGEMBEDDEDDEFAULT",
```

Comment:
I'm not a big fan of the current name here. Maybe just `DEFAULTS`.

Reply:
Done